src/main/java/exception/InvalidMoneyAmountException.java

```diff
@@ -0,0 +1,3 @@
+package exception;
+
+public class InvalidMoneyAmountException extends Exception { }
```

Comment:
Schön wäre, wenn man noch einen zusätzliche Message angeben könnte. Die Exception wird für zu viel oder zu wenig oder auch für 0 verwendet. Wenn man die Exception fängt, weiss man nicht welcher Fall es genau ist. Gute Exception bzw. Logs sind in den produktiven Systemen wichtig, damit man ohne debugging den Fehler finden kann.